src/modules/clark/learning-object-module/learning-objects.routes.ts

```diff
@@ -218,6 +218,26 @@ export const LEARNING_OBJECTS_ROUTES: ProxyRoute[] = [
         path: "/admin/learning-objects/:learningObjectNames/multiple",
         auth: true,
     },
+    /**
+     * Submissions routes
+     */
+    {
+        method: HTTPMethod.GET,
+        path: "/learning-objects/:learningObjectId/submissions",
+        target: envConfig.getUri(CLARK_SERVICE_URI),
+        auth: true,
+    },
+    {
+        method: HTTPMethod.POST,
+        path: "/learning-objects/:learningObjectId/submissions",
+        target: envConfig.getUri(CLARK_SERVICE_URI),
+        auth: true,
+    },
+    {
+        method: HTTPMethod.DELETE,
+        path: "/users/:userId/learning-objects/:learningObjectId/submissions",
+        auth: true,
+    },
```

Comment:
Just curious, was there a reason why these were moved within the file?

Reply:
The GET "/learning-objects/:learningObjectId/submissions" had a route conflict with GET "/learning-objects/:username/:learningObjectName". Moving the submissions routes above that route was done to solve that issue.